Sources/HipMobileUI/Views/BottomSheetView.cs

```diff
@@ -210,7 +210,7 @@ private void RemoveGestureRecognizer(View view, GestureRecognizer recognizer)
             Rectangle bottomSheetRect = new Rectangle
             {
                 Left = 0,
-                Top = Height * 0.9,
+                Top = Height * 0.875,
                 Size = new Size (Width, Height * 0.1)
```

Comment:
I guess you're right. I wasn't sure because on iOS everything looks fine, on Android there were some graphical "bugs", but I thought it's because of genymotion. I'll correct it after the meeting ;)